# HG changeset patch
# User Rik <rik@octave.org>
# Date 1413606825 25200
#      Fri Oct 17 21:33:45 2014 -0700
# Node ID 57147e9090424944e926ea72bdbc4e8cfa31bcbd
# Parent  30a9df3770aaec63c704e9211aa18df294f08697
copyfile.m: Fix failing BIST test.

* copyfile.m: Fix failing BIST test.

diff --git a/scripts/miscellaneous/copyfile.m b/scripts/miscellaneous/copyfile.m
--- a/scripts/miscellaneous/copyfile.m
+++ b/scripts/miscellaneous/copyfile.m
@@ -59,17 +59,18 @@ function [status, msg, msgid] = copyfile
     cmd_force_flag = "-f";
   endif
 
   ## Input type check.
   if (ischar (f1))
     f1 = cellstr (f1);
   elseif (! iscellstr (f1))
     error ("copyfile: F1 must be a string or a cell array of strings");
-  elseif (! ischar (f2))
+  endif
+  if (! ischar (f2))
     error ("copyfile: F2 must be a string");
   endif
 
   if (nargin == 3 && strcmp (force, "f"))
     cmd = [cmd " " cmd_force_flag];
   endif
 
   ## If f1 has more than 1 element then f2 must be a directory
